tiled/adapters/zarr.py

```diff
@@ -365,9 +378,12 @@ def from_catalog(
         /,
         **kwargs: Optional[Any],
     ) -> Union[ZarrGroupAdapter, ArrayAdapter]:
-        zarr_obj = zarr.open(
-            path_from_uri(data_source.assets[0].data_uri)
-        )  # Group or Array
+        parsed = urlparse(data_source.assets[0].data_uri)
```

Comment:
Take a look at what `SQLAdapter` does here. If this is bucket storage, we need to get the credentials for this bucket, similarly to how `SQLAdapter` does. The credentials are _not_ in the database, but they are in the config, and we need to match the URL of the bucket store with one in the config.

Reply:
https://github.com/bluesky/tiled/blob/f3331ef90de056e671ba5acb0a80350c473a696d/tiled/adapters/sql.py#L71-L74